<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureCameraIndividual</class>
 <widget class="QWidget" name="ConfigureCameraIndividual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>123</height>
   </rect>
  </property>
  <property name="cameraId" stdset="0">
   <number>-1</number>
  </property>
  <property name="cameraTitle" stdset="0">
   <string>NULL</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>NULL</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="camera_mode_label">
          <property name="text">
           <string>Camera Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="camera_mode_combobox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="image_path"/>
        </item>
        <item>
         <widget class="QPushButton" name="image_browse_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
